python/dgl/frame.py

```diff
@@ -189,44 +227,41 @@ def __repr__(self):
 class Frame(MutableMapping):
     """The columnar storage for node/edge features.
 
-    The frame is a dictionary from feature fields to feature columns.
+    The frame is a dictionary from feature names to feature columns.
     All columns should have the same number of rows (i.e. the same first dimension).
 
     Parameters
     ----------
     data : dict-like, optional
         The frame data in dictionary. If the provided data is another frame,
         this frame will NOT share columns with the given frame. So any out-place
-        update on one will not reflect to the other. The inplace update will
-        be seen by both. This follows the semantic of python's container.
-    num_rows : int, optional [default=0]
+        update on one will not reflect to the other.
+    num_rows : int, optional
         The number of rows in this frame. If ``data`` is provided and is not empty,
         ``num_rows`` will be ignored and inferred from the given data.
     """
-    def __init__(self, data=None, num_rows=0):
+    def __init__(self, data=None, num_rows=None):
```

Comment:
> Can `data` and `num_rows` be both None?

Not at the moment but it could be when later we create a frame using `Frame()` and append another frame to it.

> Would it cause problems?

No

Reply:
But when would we use `Frame()`?  When we don't have any data and the number of nodes/edges is 0?  If that's the only case then I would rather explicitly specify the number of nodes to be 0.

I was asking this because previously I have seen bugs in subgraph creation where the frame is incorrectly created with `Frame()` and therefore features cannot be assigned later.

A minor issue though.